rustworkx/rustworkx.pyi

```diff
@@ -192,6 +192,7 @@ def number_connected_components(graph: PyGraph, /) -> int: ...
 def number_weakly_connected_components(graph: PyDiGraph, /) -> bool: ...
 def node_connected_component(graph: PyGraph, node: int, /) -> set[int]: ...
 def strongly_connected_components(graph: PyDiGraph, /) -> list[list[int]]: ...
+def condensation(graph: PyDiGraph, /, sccs=None) -> PyDiGraph: ...
```

Comment:
```suggestion
def condensation(graph: PyDiGraph, /, sccs: list[int] | None=...) -> PyDiGraph: ...
```
You don't specify the value here, just that the argument is optional because there is already a default